__tests__/comments.tests.js

```diff
@@ -164,3 +165,29 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("DELETE /api/comments/:comment_id", () => {
+  test("DELETE - status: 204 - deletes the given comment by comment id", () => {
+    const commentId = 1;
+    return request(app).delete(`/api/comments/${commentId}`).expect(204);
```

Comment:
You could assert/make sure that the comment has been deleted by making a query for that comment in a .then after the request eg `.expect(204).then( query for removed comment  )`

You'd have to think what should come back from the database if the delete request has been successful and put that in an assertion.

Just a nice to have -